test/remote/gateways/remote_paysafe_test.rb

```diff
@@ -148,6 +148,20 @@ def test_successful_purchase_with_airline_details
     assert_equal 'F', response.params['airlineTravelDetails']['tripLegs']['leg2']['serviceClass']
   end
 
+  def test_successful_purchase_with_truncated_address
+    options = {
+      billing_address: {
+        address1: "This is an extremely long address, it is unreasonably long and we can't allow it.",
+        address2: "This is an extremely long address2, it is unreasonably long and we can't allow it.",
+        city: 'Lake Chargoggagoggmanchauggagoggchaubunagungamaugg',
+        state: 'NC',
```

Comment:
Is this a lake in Australia? I feel like they'd have a lake named something like this. And if that's the case, how dare you truncate it?!?!